sdk/core/core-rest-pipeline/src/util/tokenCycler.ts

```diff
@@ -159,7 +172,15 @@ export function createTokenCycler(
     if (!cycler.isRefreshing) {
       // We bind `scopes` here to avoid passing it around a lot
       const tryGetAccessToken = (): Promise<AccessToken | null> =>
-        credential.getToken(scopes, getTokenOptions);
+        credential.getToken(
+          scopes,
+          authenticationOptions
+            ? {
+                ...getTokenOptions,
+                authenticationOptions
+              }
+            : getTokenOptions
+        );
```

Comment:
This seems needlessly complicated. Spreading `undefined` is a no-op, so you can do something simpler:

```suggestion
        const options = { ...getTokenOptions, authenticationOptions };
        credential.getToken(scopes, options);
```